### PR TITLE
Update ansible to Open Enclave 0.18.4

### PR DESCRIPTION
**Pull request by achamayou**
To be tagged as `ccf_ci_image/oe-0.18.4`, once merged.

### CODE REVIEW

**Comment by jumaffre** [comment]
Is this out yet? It doesn't seem to be listed on https://github.com/openenclave/openenclave/releases?

**Comment by achamayou** [comment]
@jumaffre it's a Draft release, and while I seem to have access, it is indeed not public unfortunately.

**Comment by achamayou** [comment]
Available now: https://github.com/openenclave/openenclave/releases/tag/v0.18.4

File: getting_started/setup_vm/roles/openenclave/vars/common.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-oe_ver: "0.18.2"
+oe_ver: "0.18.4"
```
Comment (jumaffre) [inline getting_started/setup_vm/roles/openenclave/vars/common.yml:1]: Update changelog too?

**Review by jumaffre — APPROVED** [review]

File: getting_started/setup_vm/roles/openenclave/vars/common.yml (hunk 1 above)
Comment (achamayou) [reply getting_started/setup_vm/roles/openenclave/vars/common.yml:1]: Will do in the PR that actually changes the cmake, deb and yaml files.

**Pull request merged**